package.json

```diff
@@ -60,6 +62,7 @@
     "fxa-geodb": "1.0.4",
     "fxa-js-client": "1.0.7",
     "fxa-mustache-loader": "0.0.2",
+    "fxa-pairing-channel": "github:mozilla/fxa-pairing-channel#v1.0.1",
```

Comment:
Should we look at publishing this officially to npm?

Reply:
I was thinking about, I think we could! should work out the same way as the fxa-crypto-relier

Reply:
I don't see any downside, and it would make this neater, so 👍

Reply:
👍